src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            StringBuilder resultOf3 = new StringBuilder();
+            resultOf3.append(i + 1);
+            resultOf3.append(": ");
+            resultOf3.append(lottery.getRandomBall().getNumber());
+            resultOf3.append(lottery.getRandomBall().getColor());
```

Comment:
Each call to `lottery.getRandomBall()` generates a new random ball, which means you're getting the number and color from two different balls. You should call `getRandomBall()` once per iteration and store the result in a variable to use it for both the number and color.